src/element/QueryTable/component/QueryTableParam.svelte

```diff
@@ -138,80 +111,69 @@
 		height: 0px;
 		margin-top: 4px;
 	}
-
 	[class^="param-"] {
 		align-self: center;
 		margin-bottom: 2px;
 	}
-
-	:global(.published:not(.changed)) {
-		.param-label {
-			color: var(--ve-color-dark-text);
-		}
-	}
-
 	.param-label {
 		color: var(--ve-color-light-text);
 		margin-right: 6em;
 		font-size: 14px;
 	}
-
 	.param-values {
 		color: var(--ve-color-dark-text);
 		font-size: 13px;
 		padding: 1px 2px 1px 2px;
-
-		--height: 24px;
```

Comment:
this too

Reply:
fixed line deletions in the most recent commit